engine/synthesis_engine.cc

```diff
@@ -470,8 +470,8 @@ void SynthesisEngine::create_one_accent_list(std::vector<long> &accent_list, Nap
         Napi::Value mora = moras[i];
         Napi::Object mora_object = mora.As<Napi::Object>();
         long value;
-        if ((int)i == point || (point < 0 && (int)i == moras.Length() - point)) value = 1;
-        else value = 1;
+        if ((int)i == point || (point < 0 && (int)i == moras.Length() - 1)) value = 1;
+        else value = 0;
```

Comment:
else 時も`value`が`1`になっているのは完全にミスですね...
こちらも修正ありがとうございます！
ただ一点、`point`がマイナス値を取るとき、pythonのlistなどに対するindexと同じような動作をさせたくてこのようなコードを書いたのですが、`point`がマイナス時は`moras.Length() + point`の方が正しいことに気づきました！
出来れば汎用的なコードにしたいので、以下のように修正させてください！
```suggestion
        if ((int)i == point || (point < 0 && (int)i == moras.Length() + point)) value = 1;
        else value = 0;
```